src/schema/artist/current.js

```diff
@@ -10,25 +10,36 @@ const CurrentEventType = new GraphQLObjectType({
     image: {
       type: Image.type,
     },
-    headline: {
+    status: {
       type: GraphQLString,
+      description: "The state of the event",
     },
-    subHeadline: {
+    partner: {
       type: GraphQLString,
+      description: "Name of the partner associated to the event",
+    },
+    details: {
+      type: GraphQLString,
+      description: "Location and date of the event if available",
     },
     name: {
       type: GraphQLString,
+      description: "Name of the event",
+    },
+    page: {
```

Comment:
We've been using `href` generally for these.

Reply:
Cool, I'll update that.